ruby/neo4j/driver/internal/async/pool/netty_channel_pool.rb

```diff
@@ -0,0 +1,56 @@
+module Neo4j::Driver
+  module Internal
+    module Async
+      module Pool
+        class NettyChannelPool
+          # Unlimited amount of parties are allowed to request channels from the pool.
+          MAX_PENDING_ACQUIRES = java.lang.Integer::MAX_VALUE
+
+          # Do not check channels when they are returned to the pool.
+          RELEASE_HEALTH_CHECK = false
+
+          attr_reader :delegate, :closed, :id, :close_future
```

Comment:
only `id` was exposed with reader, in original class unless I am missing something here.

```suggestion
          attr_reader :id
```